lib/nimble_template/templates/variants/phoenix/template.ex

```diff
@@ -40,6 +40,7 @@ defmodule NimbleTemplate.Templates.Phoenix.Template do
     |> Addons.Mimic.apply()
     |> Addons.Faker.apply()
     |> Addons.Git.apply()
+    |> Addons.TestInteractive.apply()
```

Comment:
Should we add this addon into the mix variant too @byhbt 🤔

Reply:
yeah, i think that totally makes sense to have in the `mix` variant, because we also need to run tests there. Except for one thing that in the `mix` variant, the generated template does not have config. 
I updated here: https://github.com/nimblehq/elixir-templates/pull/247/commits/911faa558c4937349e2fbcb4982c09e3d5fb04c6